nscplugin/src/main/scala/scala/scalanative/nscplugin/NirGenExpr.scala

```diff
@@ -1618,14 +1902,14 @@ trait NirGenExpr { self: NirGenPhase =>
         }
       val args = genMethodArgs(sym, argsp)
       val method =
-        if (isImplClass(owner) || statically || owner.isStruct || owner.isExternModule) {
+        if (isImplClass(owner) || statically || owner.isStruct || owner.isExternModule || self.ty == nir.Type.Ptr) {
```

Comment:
`self.ty == nir.Type.Ptr` Should this have a name like `isPointer` or something?